charts/cloudserver-front/templates/dashboard.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sample-grafana-dashboard
+  labels:
+    app: {{ template "cloudserver-front.name" . }}
+    chart: {{ template "cloudserver-front.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    grafana-dashboard: "true"
```

Comment:
Maybe this label should also depend on the release name, otherwise we'll end up with dashboards from different releases shared across various Grafana instances.